app/views/services/govbox/register_step2.html.erb

```diff
@@ -7,27 +7,62 @@
         <h1 class="form-title">Údaje štatutára</h1>
         <p class="form-subtitle lead">Zadajte údaje štatutára (napríklad konateľa s.r.o.), ktorý podpíše splnomocnenie na prístup k schránke.</p>
 
-        <div class="row">
-          <div class="col-md-4 form-group">
-            <%= label_tag :given_name, 'Meno' %>
-            <%= text_field_tag :given_name, params[:given_name], class: 'form-control input-lg', required: true %>
+        <% if @statutory_entries %>
+          <h6 style="margin: 30px 0;">Vyberte štatutára zo zoznamu:</h6>
+          <div class="row">
+            <div class="col-md-12">
+              <% @statutory_entries.each_with_index do |entry, index| %>
```

Comment:
naco je tu `index`?

Reply:
chcel som, nech sme HTML korektni a nech mame pre elementy nerovnake ID, bez toho myslim budu mat vsetky `id="statutory_entry"` . ci sa mylim?

Reply:
aha, ok, nechajme to tak